src/index.js

```diff
@@ -88,6 +92,9 @@ export default function alias(options = {}) {
         return filePath + '.js';
       }
 
+      const nodeModuleFile = resolveNodeModules(updatedId);
+      if (nodeModuleFile) return nodeModuleFile;
```

Comment:
I have some suggestions:
1. `process.cwd()` cannot use as project root. If people don't invoke `node` from project root, your code will wrong.
2. There may be other rollup plugins that can deal with module id. So resolving using node_modules folder as fallback can cause compatibility issue.

So you can try to add two config options: projectRoot and nodeModulesFallback
```
rollup({
  entry: './src/index.js',
  plugins: [alias({
    vue: 'vue/dist/vue.esm.js'
  }, projectRoot, nodeModulesFallback)],
});
```
projectRoot is used to indicate project root directory.
nodeModulesFallback default is false for backwards compatibility.

I think using `alias` in this way is also convenient:
```
rollup({
  entry: './src/index.js',
  plugins: [alias({
    vue: path.resolve('PROJECT_ROOT/node_modules', './vue/dist/vue.esm.js')
  })],
});
```